test/acceptance/tests/controller/controller_test.go

```diff
@@ -41,7 +41,7 @@ func TestController(t *testing.T) {
 				"controller.enabled":    "true",
 				"connectInject.enabled": "true",
 				// todo: remove when 1.9.0 is released.
-				"global.image": "hashicorpdev/consul",
+				"global.image": "hashicorp/consul-enterprise:1.9.0-ent-beta2",
```

Comment:
This was easier than setting the license values to `""`

Reply:
Sorry, don't understand this. Not a big deal though because this will all be removed with 1.9.0.

Reply:
Sorry, this was confusing! This is because if we set ent license flags globally through test flags and use OSS image, the helm install will fail. One option is to keep OSS image and set ent license flags to `""`, another option is to do what I did. It was simpler to switch to the ent image since, as you said, it doesn't matter and it was less code to write 😄 .